### PR TITLE
changelog: 8.15.2 release

### PR DESCRIPTION
**Pull request by inge4pres**
Update changelog for 8.15.2 release

### CODE REVIEW

File: changelogs/8.15.asciidoc [hunk 1]
```diff
@@ -1,9 +1,21 @@
...
+[float]
+[[apm-release-notes-8.15.2]]
+=== APM version 8.15.2
+
+https://github.com/elastic/apm-server/compare/v8.15.1\...v8.15.2[View commits]
+
+[float]
+==== Bug fixes
+
+- Fix broken APM Agents configuration cache {pull}13958[13958]
```
Comment (carsonip) [inline changelogs/8.15.asciidoc:17]:
```suggestion
- Fix broken APM Agents configuration cache when there are more than 100 agent config entries {pull}13958[13958]
```

File: changelogs/8.15.asciidoc [hunk 2]
```diff
@@ -1,9 +1,21 @@
 [[apm-release-notes-8.15]]
 == APM version 8.15
 
+* <<apm-release-notes-8.15.2>>
 * <<apm-release-notes-8.15.1>>
 * <<apm-release-notes-8.15.0>>
 
+[float]
+[[apm-release-notes-8.15.2]]
+=== APM version 8.15.2
```
Comment (carsonip) [inline changelogs/8.15.asciidoc:10]: seems to be missing bugfix https://github.com/elastic/apm-data/pull/346 and feature https://github.com/elastic/apm-data/pull/339

Comment (inge4pres) [reply changelogs/8.15.asciidoc:10]: Thanks I was not aware of these, didn't spot them in the compare

Comment (inge4pres) [reply changelogs/8.15.asciidoc:10]: Updated!

**Review by carsonip — APPROVED** [review]
lgtm, thanks

**Pull request merged**